src/Module/Admin/Site.php

```diff
@@ -471,7 +478,7 @@ protected function content(array $request = []): string
 			'$maximagesize'     => ['maximagesize', DI::l10n()->t('Maximum image size'), DI::config()->get('system', 'maximagesize'), DI::l10n()->t('Maximum size in bytes of uploaded images. Default is 0, which means no limits.
 													The value of <code>upload_max_filesize</code> in your <code>PHP.ini</code> needs be set to at least the desired limit.
 													Currently <code>upload_max_filesize</code> is set to %s (%sB)', Strings::getBytesFromShorthand(ini_get('upload_max_filesize')), ini_get('upload_max_filesize')),
-													'1 kB = 1 KiB = 1024 Bytes'],
+													'', '', '', '\d+(?:\s*[kmg])?'],
```

Comment:
You can use the sixth field (`$field.5`) for this.
```suggestion
													'', 'pattern="\d+(?:\s*[kmg])?"'],
```